setup.py

```diff
@@ -14,7 +14,7 @@ def build_extensions(self):
             pass
         build_ext.build_extensions(self)
 
-ext = Extension('mc', sources=["mamonca/mc.pyx"], language="c++", extra_compile_args=['-fopenmp'], extra_link_args=['-lgomp'])
+ext = Extension('mc', sources=["mamonca/mc.pyx", "mamonca/cMC.cpp", "mamonca/cMC.h", "mamonca/cMC.pxd"], language="c++", extra_compile_args=['-fopenmp'], extra_link_args=['-lgomp'])
```

Comment:
I tried to compile my code with this change, but it failed to load `cMC.h`. Did it work for you?